kayenta-signalfx/src/integration-test/java/com/netflix/kayenta/config/SignalFxMockServiceReportingConfig.java

```diff
@@ -119,14 +119,16 @@ public void start() {
 
     metricsReportingStartTime = Instant.now();
 
-    // Wait for the mock services to send data, before allowing the tests to run
-    try {
-      long pause = TimeUnit.MINUTES.toMillis(CANARY_WINDOW_IN_MINUTES) + TimeUnit.SECONDS.toMillis(15);
-      log.info("Waiting for {} milliseconds for mock data to flow through SignalFx, before letting the integration tests run", pause);
-      Thread.sleep(pause);
-    } catch (InterruptedException e) {
-      log.error("Failed to wait to send metrics", e);
-      throw new RuntimeException(e);
+    if (Boolean.valueOf(System.getProperty("block.for.metrics", "true"))) {
+      // Wait for the mock services to send data, before allowing the tests to run
+      try {
+        long pause = TimeUnit.MINUTES.toMillis(CANARY_WINDOW_IN_MINUTES) + TimeUnit.SECONDS.toMillis(15);
+        log.info("Waiting for {} milliseconds for mock data to flow through SignalFx, before letting the integration tests run", pause);
+        Thread.sleep(pause);
+      } catch (InterruptedException e) {
+        log.error("Failed to wait to send metrics", e);
+        throw new RuntimeException(e);
+      }
     }
   }
 
```

Comment:
Extraneous blank line.

Reply:
Is this comment for this line?

Reply:
Hm, doesn't look like it. Please disregard.

Reply:
- [X] done